scenarios/nomad-consul-secure/README.md

```diff
@@ -51,6 +62,8 @@ You can export the required environment variables to access both Nomad and Consu
 $ eval $(shikari env -n murphy -tai consul)
 $ eval $(shikari env -n murphy -tai nomad)
 
+$ env|egrep 'CONSUL_HTTP_ADDR|NOMAD_ADDR'
+
```

Comment:
Lets filter `'CONSUL|NOMAD'` so that the user can see all the variables populated for a product. 

We may have to remove the `-i` option on lie 62 and 63 so that the CA_CERTS are respected by the clients.

Reply:
You mean it should look like : (without -i)
```
$ eval $(shikari env -n murphy -ta consul)
$ eval $(shikari env -n murphy -ta nomad)
```